.golangci.yml

```diff
@@ -15,5 +11,7 @@ linters:
 linters-settings:
   goimports:
     local-prefixes: github.com/argoproj-labs/argocd-image-updater
-service:
-  golangci-lint-version: 1.52.2
+issues:
+  exclude:
+    - ".*\\.pb\\.go"
+    - "vendor/.*"
```

Comment:
I think we should list `exclude-files` and `exclude-dirs` separately for clarity. In the above the vendor exclusion doesn't seem to include all subdirs.

```yaml
exclude-files:
  - ".*\\.pb\\.go"
exclude-dirs:
  - vendor/
```